### PR TITLE
cobra library for commandline handling

### PR DESCRIPTION
**Pull request by surajssd**
Previously flags was used for cmdline handling which is very minimal but with cobra is extensible easily.

### CODE REVIEW

**Comment by surajssd** [comment]
I will need to edit tests as well

File: pkg/types/types.go [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package types
+
+// TODO: User can put values like --port "mariadb=3306:3306"
```
Comment (surajssd) [inline pkg/types/types.go]: Here we add things we want to do like get information that we ask in interactive mode?

Comment (kadel) [inline pkg/types/types.go]: Can we delete `ConfigValues` if its not used right now?

File: pkg/cmd/root.go [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/redhat-developer/henge/pkg/types"
+)
+
+const cliLong = `
+Henge takes a docker-compose.yml file and converts it into a openshift or kubernetes artifacts,
```
Comment (rtnpro) [inline pkg/cmd/root.go]: we can accept multiple files as well, can't we?

Comment (surajssd) [inline pkg/cmd/root.go]: Yes we can https://github.com/redhat-developer/henge/pull/32/files#diff-a7f30f84a9d7d21cfedcc79a96ac9037R31

Comment (surajssd) [inline pkg/cmd/root.go]: Yes here is the code for it https://github.com/redhat-developer/henge/pull/32/files#diff-a7f30f84a9d7d21cfedcc79a96ac9037R51

File: pkg/types/types.go (hunk 1 above)
Comment (surajssd) [inline pkg/types/types.go]: Yes removed it :)

**Comment by rtnpro** [comment]
LGTM :+1:

**Comment by kadel** [comment]
Great work @surajssd :+1: , hit the magic button :wink:

**Pull request merged**